features/ooms.feature

```diff
@@ -19,6 +19,7 @@ Scenario: OOM from a single large allocation
 
 @not_symfony_2
 @not_symfony_4
+@not_symfony_7
```

Comment:
Is this because we've switched to `prod` environment? However, will we test this feature going forwards into next versions? Or do we need a debug fixture too?

Reply:
> Is this because we've switched to prod environment?

Yep, Symfony runs OOM trying to create an `OutOfMemoryError` with such small allocations

> However, will we test this feature going forwards into next versions? Or do we need a debug fixture too?

I'll be looking into if there's a way we can handle this case as well